modules/aws/src/smithy4s/aws/package.scala

```diff
@@ -32,4 +33,22 @@ package object aws {
   private[aws] def utf8String[F[_]: MonadThrow](bytes: Array[Byte]): F[String] =
     MonadThrow[F].catchNonFatal(new String(bytes, "UTF-8"))
 
+  // format: off
+  def simplify[Alg[_[_, _, _, _, _]], F[_]](service: Service[Alg]): service.Interpreter[AwsCall[F, *, *, *, *, *]] => service.FunctorInterpreter[F] = {
```

Comment:
I'm wondering if it'd be more convenient for the callers if `service` was implicit and `interpreter` was a normal function parameter.